modules/acquia_cms_tour/src/Form/AcquiaCmsDashboardBase.php

```diff
@@ -77,7 +84,11 @@ public static function create(ContainerInterface $container) {
       $container->get('state'),
       $container->get('module_handler'),
       $container->get('link_generator'),
-      $container->get('info_parser')
+      $container->get('info_parser'),
+      $container->get("config.factory"),
+      // The service `config.typed` has been introduced in Drupal Core 10.2.x.
+      // @todo Remove below condition,once we drop support for Drupal < 10.2.x.
+      $container->has('config.typed') ? $container->get('config.typed') : NULL
```

Comment:
Even with drupal <10.2 we can just use as below, no need to check for service existence, since we are already type hinted with `?TypedConfigManagerInterface` which will take care of it.
```
$container->get("config.factory"),
$container->get('config.typed')
```